BITS-App/AppShell.xaml

```diff
@@ -8,7 +8,7 @@
     Shell.FlyoutBehavior="Disabled">
 
     <ShellContent
-        ContentTemplate="{DataTemplate views:PostPage}"
+        ContentTemplate="{DataTemplate views:StaffProfilePage}"
```

Comment:
nothing to change here, but just a note for other: we will be adding navigation soon